include/envoy/ssl/connection.h

```diff
@@ -129,6 +130,15 @@ class ConnectionInfo {
    *         connection.
    **/
   virtual const std::string& tlsVersion() const PURE;
+
+  /**
+   * Retrieves the value of a specific X.509 extension from the peer cert, if a peer cert exists
+   * and it contains the specified extension.
+   * @param extension_name name of extension to look up
+   * @return absl::optional<std::string> the value of the extension, if it exists.
```

Comment:
What is the format of the value? Is it a human-readable string? An ASN1 representation of something?

Reply:
just the raw bytes contained within the ASN.1 object, ill update the comment